source/operatingsystems/linux/mysql/mysql_ssh_tunneling.md

```diff
@@ -0,0 +1,89 @@
+     When entering passwords, MySQL Workbench offers users the ability to store passwords in a Vault, which is encrypted on your machine. See [MySQL - The Password Storage Vault](https://dev.mysql.com/doc/workbench/en/wb-mysql-connections-vault.html) for more information.
+```
+
+### Testing the connection
+- Once you've entered all the above parameters, click `Test Connection`.
+- You may receive a prompt about the fingerprint not being recognised. If this is the first time connecting to the server and you're confident that the details you've entered are correct, click **Yes**. 
+
+This should return **Successfully made the MySQL connection**. Click **Ok**, and then **Ok** again to save the configuration. If this is unsucessful due to incorrect MySQL credentials, check that the user you're connecting with is set with the host `localhost`.
+
+## SSH Tunnelling with Other Clients (Windows)
+While most MySQL clients support SSH tunnelling, some clients do not. Using PuTTY, we can create an SSH Tunnel systemwide, so that you can connect to MySQL on localhost:3306, which will route the traffic to the server. To do this, follow the following steps: 
+
+- If you don't already have PuTTY, you can download it [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Download the 32/64-bit MSI (most modern operating systems are 64-bit, so if unsure, try this first). Open the downloaded .exe and go through the installation steps.
+ - Once installed, open PuTTY. Below are the values to enter, along with their description. 
+
+ ### Configuring PuTTY with SSH Tunnelling
+
+ #### Session
+ On PuTTY, on the left side, you can see the different categories for values. Start off by selecting Session, and fill in the values below: 
+
+ | Value | Description |
+|-|-|
+| SSH Hostname | Your SSH username followed by the IP address of the server you're connecting to. For example, `root@1.2.3.4` |
+| SSH Port | The default SSH port for UKFast servers is 2020, otherwise, try 22. |
+| Saved Sessions | Enter a name to uniquely identify this connection.  |
+
+ #### Tunnels
+ On the left menu, scroll down and select Tunnels (under Connection > SSH > Tunnels). This is where we will configure the SSH tunnel.   
+
+ | Value | Description |
+|-|-|
+| Source Port | 3306 is the default MySQL port. |
+| Destination | localhost:3306 |
+| Radio Button: Local | Local forwards a local port to a remote server. |
+| Radio Button: Auto | Keep this on Auto  | 
+  
+Click `Add`.
+
+#### Optional: Key Based Authentication
+If you're using SSH Keys to authenticate, go to Connections > SSH > Auth. Here you can browse for your SSH private key. 
+
+### Saving and testing your configuration
+- Once you've done the above steps, go back to Session and click `Save`. This saves your configuration under the name you specified under Saved Sessions. 
+- Whenever you now open putting, you will be able to load your configuration by selecting the Saved Sessions name and clicking `Load`. 
```

Comment:
`PuTTY`?
'Load'  and 'Saved Sessions' in bold instead as something/someplace you are clicking.